### ISSUE TITLE
When in split mode, fullscreen does weird things

### ISSUE THREAD

**Issue by ByJumperX4** [Jelmerro/Vieb#143]
**Checklist**
- [x] I searched the `:help` documentation using `/` for helpful information
- [x] I have read the [frequently asked questions](https://github.com/Jelmerro/Vieb/blob/master/FAQ.md)
- [x] I did not find any similar issues in the [issue tracker](https://github.com/Jelmerro/Vieb/issues)

**Describe the bug**
When in split mode, having a video in fulscreen will fuck up. The focused page will remain ok, but the others will become gray and unresponsive to keyboard

**To Reproduce**
Enable split mode with `:split` or `:vsplit`, open a web page containing a video, enable fullscreen for the video.

https://user-images.githubusercontent.com/48773926/113423202-52679c00-93ce-11eb-8e13-11ef04305880.mp4

**Expected behavior**
Fullscreen should cover the others web pages, or it should stay in its part.

**Specs**
 - OS: Funtoo Linux w/ dwm 
 - Version: latest master compiled from source

**Comment by Jelmerro** [comment]
Thanks for this detailed bug report (and other issues as well :smile:). I was able to reproduce it easily, not sure how I missed this, but it's been like this ever since I started using a separate pagelayout manager for splitting the page. It seems like the styling applied to position the webview is overriding the fullscreen styling. Sadly, I can only override the style attribute by using `!important` when in fullscreen, but this can and will be fixed soon.
